solutions/kubernetes/io-testing.sh

```diff
@@ -22,6 +22,41 @@ set -euo pipefail # exit on failures
 
 source /var/tmp/functions.sh
 
+add_separator Setup awscli.
+
+echo -e "\nInstall awscli:-\n"
+pip3 install awscli
+pip3 install awscli-plugin-endpoint
+
+if ! which aws; then
+  add_separator "AWS CLI installation failed"
+  exit 1
+fi
+
+access_key=$(kubectl get pods | grep cortx-server- | cut -d " " -f1 | head -1 | xargs -I{} kubectl exec -t {} -c cortx-rgw -- cat /etc/cortx/cluster.conf | grep auth_admin | head -2 | tail -1 | cut -d ":" -f2 | sed -e 's/^[[:space:]]*//')
+secret_key=$(cat /root/deploy-scripts/k8_cortx_cloud/solution.yaml | grep s3_auth_admin_secret: | cut -d ":" -f2)
```

Comment:
Seems `s3_auth_admin_secret` value is not as per https://github.com/Seagate/cortx-k8s/blob/integration/k8_cortx_cloud/solution.yaml#L9 in RE scripts. 
Need to correct it at https://github.com/Seagate/cortx-re/blob/main/solutions/kubernetes/cortx-deploy-functions.sh#L67

Reply:
This value is as per Release v0.1.0 (ldapadmin), there its different from integration (cortxadmin). Since deployments are done with Release v0.1.0, for integration branch manual deployment can be done.